src/main/java/vazkii/psi/common/block/base/ModBlocks.java

```diff
@@ -116,7 +113,7 @@ public static void register(RegisterEvent evt) {
 
 		evt.register(ForgeRegistries.Keys.MENU_TYPES, helper -> {
 			helper.register(ForgeRegistries.BLOCKS.getKey(cadAssembler), containerCADAssembler);
-			DistExecutor.safeRunWhenOn(Dist.CLIENT, () -> () -> {
+			DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> () -> {
```

Comment:
Is unsafe necessary?

Reply:
Yes
Caused by: java.lang.RuntimeException: Unsafe Referent usage found in safe referent method
The olde runWhenOn function just wraps unsafeRunWhenOn.


@Deprecated
public static void runWhenOn(Dist dist, Supplier<Runnable> toRun) {
    unsafeRunWhenOn(dist, toRun);
}
https://forums.minecraftforge.net/topic/96761-1165-properly-using-distexecutor-with-arguments/